setup.cfg

```
@@ -1,6 +1,6 @@
 [metadata]
 name = heroicons
-version = 1.8.0
+version = 2.0.0
```

My release script will bump this
```suggestion
version = 1.8.0
```